src/main/kotlin/com/manimdsl/Compiler.kt

```diff
@@ -30,27 +33,60 @@ private fun compile(filename: String) {
     val executor = ASTExecutor(abstractSyntaxTree, symbolTable, file.readLines())
 
     var state: Pair<Boolean, List<ManimInstr>>
-
     do {
          state = executor.executeNextStatement()
     } while (!state.first)
 
     val writer = ManimProjectWriter(ManimWriter(state.second).build())
 
-    writer.createPythonFile("test.py")
-    writer.generateAnimation("test.py")
+    if(output !== null) println("Writing file to $output")
+    val outputFile = writer.createPythonFile(output)
```

Comment:
What happens here if `output` is null?

Reply:
It creates a temp file as they didn't ask for the python file out